web3.md

```diff
@@ -580,30 +580,76 @@ WEB3 JSON RPC
 **TODO**: You're supposed to be able to overwrite a pending tx by using its same "from" and "nonce" values
 
 ```k
-    syntax KItem ::= "loadTX" JSON
- // ------------------------------
-    rule <k> loadTX J => signTX !TXID ACCTADDR ... </k>
-         <txPending> .List => ListItem( !TXID ) ... </txPending>
+    syntax KItem ::= "createTX" Int JSON
+ // ------------------------------------
+    rule <k> createTX TXID J => loadTX TXID J ~> signTX TXID ACCTID ... </k>
+         <txPending> (.List => ListItem(TXID)) ... </txPending>
          <account>
-           <acctID> ACCTADDR </acctID>
-           <nonce> ACCTNONCE => ACCTNONCE +Int 1 </nonce>
+           <acctID> ACCTID </acctID>
+           <nonce>  ACCTNONCE </nonce>
            ...
          </account>
          <messages>
-           (.Bag => <message>
-             <msgID> !TXID:Int </msgID>
-             <txNonce>    #if isString( #getJSON("nonce",    J) ) #then #parseHexWord(#getString("nonce",J))    #else ACCTNONCE  #fi </txNonce>
-             <to>         #if isString( #getJSON("to",       J) ) #then #parseHexWord(#getString("to",J))       #else 0          #fi </to>
-             <txGasLimit> #if isString( #getJSON("gas",      J) ) #then #parseHexWord(#getString("gas",J))      #else 90000      #fi </txGasLimit>
-             <txGasPrice> #if isString( #getJSON("gasPrice", J) ) #then #parseHexWord(#getString("gasPrice",J)) #else GASP       #fi </txGasPrice>
-             <value>      #if isString( #getJSON("value",    J) ) #then #parseHexWord(#getString("value",J))    #else 0          #fi </value>
-             <data>       #if isString( #getJSON("data",     J) ) #then #parseByteStack(#getString("data",J))   #else .ByteArray #fi </data>
-             ...
-           </message>)
-         ...
-         </messages>
-         <gasPrice> GASP </gasPrice>
-      requires #parseHexWord(#getString("from",J)) ==Int ACCTADDR
+            ( .Bag
+           => <message>
+                <msgID> TXID </msgID>
+                <txGasPrice> 1         </txGasPrice>
+                <txNonce>    ACCTNONCE </txNonce>
+                <txGasLimit> 90000     </txGasLimit>
+                ...
+              </message>
+            )
+          ...
+          </messages>
+      requires #parseHexWord(#getString("from", J)) ==Int ACCTID
```

Comment:
We should be structurally matching on `ACCTID` instead of constraining it in the constraint like we're doing here.

This requires cycling through all the accounts and grabbing the one which has this constraint, but matching on it directly lets us just do a log-n lookup.

Reply:
`createTX` now takes an `address` and the `JSON` message. By applying the rule `createTX #parseHexWord(#getString("from", J)) J` we should be matching on `ACCTID`.